packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -192,38 +174,61 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
     );
   };
 
+  private _selectedKey(): string {
+    if (this._isKeyValid(this.props.selectedKey)) {
+      return this.props.selectedKey!;
+    }
+    if (this._isKeyValid(this.state.selectedKey)) {
+      return this.state.selectedKey;
+    }
+    if (this._links.length) {
+      return this._links[0].itemKey!;
+    }
+    return '';
```

Comment:
Not a blocker, but what are your thoughts regarding returning `undefined` here rather than an empty string? It seems to make the intent / result clearer. It _should_ work for the cases below, see:

```js
var strs = ['a', 'b']
undefined
strs[undefined]
undefined
strs['']
undefined
```

Reply:
Yeah - this is a good idea. Thanks!